### DISCUSSION TITLE
Problem with security protocol

### DISCUSSION THREAD

**Discussion by ShadowMasterRu** [Tyrrrz/DiscordChatExporter#610] [Q&A] [answered]
Hello. I get this error message when try to start DCE
![170621 23-58-34](https://user-images.githubusercontent.com/86084488/122483073-718fc600-cfda-11eb-820d-3b746e7d7e50.png)
Nothing was changed in my OS (Win 7 x64) from the last succesful use of DCE. What exactly I must install or edit to solve this problem?
P.S.: Installing NET 3.1 and 5 Runtime didn't change anything

**Comment by Tyrrrz** [comment, marked as answer]
This bug should be fixed in latest master. Try a CI build from here: https://github.com/Tyrrrz/DiscordChatExporter/actions/runs/947535713

**Reply by ShadowMasterRu** [reply]
Oh, thank you. And, of course, thanks for very useful soft!
